Sources/SwiftIfConfig/BuildConfiguration.swift

```diff
@@ -9,6 +9,7 @@
 // See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
 //===----------------------------------------------------------------------===//
+import SwiftSyntax
```

Comment:
Could you add a newline between the copyright header and the import for consistency with the other files?

Reply:
Smells like a linter rule to me, but okay